### PR TITLE
Try to debug #61

### PR DESCRIPTION
**Pull request by zklaus**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] ~Reset the build number to `0` (if the version changed)~
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by zklaus** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -58,6 +58,7 @@ outputs:
       files:
         - mpiexec.sh
         - tests/helloworld.sh
+        - mpiexec
```
Comment (dalcinl) [inline recipe/meta.yaml:61]: This is wrong. We do not have a file named "mpiexec" in the recipe directory. What are you trying to do?

Comment (dalcinl) [reply recipe/meta.yaml:61]: FYI, this is from my home workstation, running MPICH from Fedora 34 (system package, not conda forge)

```
$ mpichversion | head -n 3
MPICH Version:    	3.4.1
MPICH Release date:	Fri Jan 22 14:17:48 CST 2021
MPICH Device:    	ch3:nemesis

$ mpiexec
Segmentation fault (core dumped)
```

Indeed, a bare `mpiexec` invocation without args segfaults. This is most likely a bug in MPICH and it is not related conda-forge build/packaging.

@hzhou FYI.

Comment (zklaus) [reply recipe/meta.yaml]: Yeah, I got the same thing. If we can conclude that this is simply a bug in mpich, perhaps we can just close #61, possibly with a link to the relevant mpich ticket.

**Comment by zklaus** [comment]
With #61 considered an upstream bug, this PR is obsolete. Thanks for the help.

**Pull request closed without merging**